features/step_definitions/files_steps.rb

```diff
@@ -30,3 +30,10 @@
   expect(uncommitted_files).to eql [file_name]
   expect(IO.read file_name).to eql content
 end
+
+
+Then(/^my uncommitted file "(.+?)" is still stashed away$/) do |file_name|
+  expect(uncommitted_files).to_not include file_name
```

Comment:
Should this check if the file is actually in the stash rather than checking that it's just not uncommitted? Maybe something like `git stash show` http://stackoverflow.com/questions/10725729/git-see-whats-in-a-stash-without-applying-stash